model_analyzer/record/metrics_manager.py

```diff
@@ -497,6 +501,42 @@ def _run_perf_analyzer(self, run_config, perf_output_writer):
 
         return aggregated_perf_records, aggregated_gpu_records
 
+    def _write_perf_analyzer_output(self,
+                                    perf_output_writer: Optional[FileWriter],
+                                    perf_analyzer: PerfAnalyzer) -> None:
+        if perf_output_writer:
+            perf_output_writer.write(
+                '============== Perf Analyzer Launched ==============\n'
+                f'Command: {perf_analyzer.get_cmd()}\n\n',
+                append=True)
+            if perf_analyzer.output():
+                perf_output_writer.write(perf_analyzer.output() + '\n',
+                                         append=True)
+
+    def _handle_unsuccessful_perf_analyzer_run(
+            self, perf_analyzer: PerfAnalyzer) -> None:
+        output_path = self._config.perf_output_path if self._config.perf_output_path else self._config.export_path
```

Comment:
Sorry for the churn. I don't think it should be variable like this, especially because now two different objects need to understand how to re-calculate this path. Too hard to document and too confusing. I misunderstood what perf_output_path was for and lead you down the wrong path.  Let's just stick with the export path